common/statsd/statsd.cc

```diff
@@ -108,6 +108,9 @@ bool StatsD::submitCounters(const CounterState &counters, string_view host, int
     }
 
     for (auto &hist : counters.counters->histograms) {
+        if (ignoredHistograms->find(hist.first) != std::string_view::npos) {
```

Comment:
... is this `->find` silently converting the C-style array to a pointer, and accessing the first element?

Let's not use C-style arrays

Reply:
as in, i think you're just doing `ignoredHistograms[0].find(hist.first)`

Reply:
ah great catch, replaced with `std::find` and `std::array`